doc/source/reference/arrays.classes.rst

```diff
@@ -154,11 +154,6 @@ NumPy provides several hooks that classes can customize:
         to have a hierarchy of subclasses that extend the behaviour. See
         :ref:`Subclassing ndarray <basics.subclassing>` for details.
 
-   .. note:: If a class defines the :func:`__array_ufunc__` method,
-      this disables the :func:`__array_wrap__`,
-      :func:`__array_prepare__`, :data:`__array_priority__` mechanism
-      described below for ufuncs (which may eventually be deprecated).
```

Comment:
Maybe someone disagrees that this comment is useless.  We don't *disable* it, you just have to `view()` as a base ndarray to avoid recursion.